src/main/java/com/fasterxml/jackson/core/exc/StreamReadException.java

```diff
@@ -52,6 +52,11 @@ protected StreamReadException(String msg, JsonLocation loc, Throwable rootCause)
         super(msg, loc, rootCause);
     }
 
+    // @since 2.15
+    protected StreamReadException(String msg) {
```

Comment:
I will merge this and similar change in `JsonParseException` separate from this PR.